mne/source_estimate.py

```diff
@@ -1911,12 +1931,18 @@ class VolSourceEstimate(_BaseVolSourceEstimate):
     Parameters
     ----------
     data : array of shape (n_dipoles, n_times) | tuple, shape (2,)
-        The data in source space. The data can either be a single array or
-        a tuple with two arrays: "kernel" shape (n_vertices, n_sensors) and
-        "sens_data" shape (n_sensors, n_times). In this case, the source
-        space data corresponds to "numpy.dot(kernel, sens_data)".
-    vertices : array
-        Vertex numbers corresponding to the data.
+        The data in source space. When it is a single array, the
+        left hemisphere is stored in data[:vertices[0]] and the right
+        hemisphere is stored in data[:-vertices[1]].
+        When data is a tuple, it contains two arrays:
```

Comment:
again this is not correct. You have here a VolSourceEstimate

Reply:
oups, not sure how this crept in ...